packages/react-instantsearch/src/connectors/connectHighlight.js

```diff
@@ -22,35 +22,41 @@ const highlight = ({ attribute, hit, highlightProperty }) =>
  * @providedPropType {function} highlight - function to retrieve and parse an attribute from a hit. It takes a configuration object with 3 attributes: `highlightProperty` which is the property that contains the highlight structure from the records, `attribute` which is the name of the attribute (it can be either a string or an array of strings) to look for and `hit` which is the hit from Algolia. It returns an array of objects `{value: string, isHighlighted: boolean}`. If the element that corresponds to the attribute is an array of strings, it will return a nested array of objects.
  * @example
  * import React from 'react';
- * import { connectHighlight } from 'react-instantsearch/connectors';
- * import { InstantSearch, Hits } from 'react-instantsearch/dom';
+ * import { InstantSearch, SearchBox, Hits } from "react-instantsearch/dom";
+ * import { connectHighlight } from "react-instantsearch/connectors";
```

Comment:
Did you do these changes manually or with prettier, since I think it should have single quotes :)

Reply:
My bad I use Prettier playground sometimes... I will update it.

Reply:
Thought so, luckily in v2 prettier will use singleQuote and trailingComma by default 🙌

Reply:
Thanks for the catch!